build.sbt

```diff
@@ -686,16 +686,29 @@ lazy val kafkaUtils = (project in utils("kafka-utils")).
   settings(itSettings()).
   settings(
     name := "nussknacker-kafka-utils",
+    libraryDependencies ++= {
+      Seq(
+        "org.apache.kafka" % "kafka-clients" % kafkaV
+      )
+    }
+  // Depends on componentsApi because of dependency to NuExceptionInfo and NonTransientException -
+  // lite kafka engine handles component exceptions in runtime part
+  ).dependsOn(commonUtils % Provided, componentsApi % Provided)
+
+lazy val kafkaComponentsUtils = (project in utils("kafka-components-utils")).
```

Comment:
please add changelog and migration guide that such a new modules is available and has classes from nussknacker-kafka-utils

Reply:
Added